packages/@react-spectrum/s2/src/TagGroup.tsx

```diff
@@ -62,7 +67,13 @@ export interface TagGroupProps<T> extends Omit<AriaTagGroupProps, 'children' | '
   /** Whether the tags are displayed in a error state. */
   isInvalid?: boolean,
   /** An error message for the field. */
-  errorMessage?: ReactNode
+  errorMessage?: ReactNode,
+  /** Limit the number of rows initially shown. This will render a button that allows the user to expand to show all tags. */
+  maxRows?: number,
+  /** The label to display on the action button.  */
+  actionLabel?: string,
+  /** Handler that is called when the action button is pressed. */
+  onAction?: () => void
```

Comment:
Do we want to change the name of this prop? `onActionButton`?
Right now our TagGroup doesn't support actions on tags, so it's not in conflict necessarily.

Reply:
I think it makes sense to change the name for consistency with other collection components. `onGroupAction` is another option.